src/main/scala/mesosphere/marathon/core/appinfo/impl/DefaultInfoService.scala

```diff
@@ -17,15 +18,14 @@ import scala.concurrent.Future
 
 private[appinfo] class DefaultInfoService(
     groupManager: GroupManager,
-    newBaseData: () => AppInfoBaseData) extends AppInfoService with GroupInfoService with PodStatusService {
-  import mesosphere.marathon.core.async.ExecutionContexts.global
+    newBaseData: () => AppInfoBaseData) extends AppInfoService with GroupInfoService with PodStatusService with StrictLogging {
 
-  private[this] val log = LoggerFactory.getLogger(getClass)
+  implicit val ec = NamedExecutionContext.fixedThreadPoolExecutionContext(8, "default-info-service")
```

Comment:
I wonder if this really has an impact if app info and the group manager have their own thread pools. I'd like to understand where we benefit from a fixed worker thread pool and where we should be using the fork join thread pool.

Reply:
I tested the scenario with one thread pool and it adds roughly 20% response time. Not that much but I think dividing them still makes sense - Dīvide et Imperā!